packages/dso-toolkit/src/components/highlight-box/highlight-box.mixins.scss

```diff
@@ -73,22 +73,13 @@
 @mixin exoticChildren() {
   .dso-tertiary:not([disabled]),
   .btn-link:not([disabled]) {
-    color: colors.$grijs-90;
-
-    &:hover,
-    &:focus {
-      &,
-      dso-icon,
-      svg.di {
-        color: colors.$grijs-60;
-      }
-    }
+    color: colors.$zwart;
```

Comment:
Bij hover op link werd de tekst lichter. Het lijkt mij dat het de bedoeling is dat niet te doen? Met deze change komt er op hover alleen een underline.

Reply:
Zeker.